users/luescher/setups/librispeech/pipeline_base_args.py

```diff
@@ -322,25 +336,25 @@ def get_triphone_args(
     triphone_recognition_args = {
         "iters": [8, 10],
         "feature_flow": feature_flow,
-        "pronunciation_scales": [6.0],
-        "lm_scales": [24.9],
+        "pronunciation_scales": [1.0],
+        "lm_scales": [25.0],
         "lm_lookahead": True,
         "lookahead_options": None,
         "create_lattice": True,
         "eval_single_best": True,
         "eval_best_in_lattice": True,
         "search_parameters": {
-            "beam_pruning": 12.0,
-            "beam-pruning-limit": 100000,
+            "beam_pruning": 15.0,
+            "beam-pruning-limit": 50000,
```

Comment:
I will do a full review later, but I can already say that 50k here is not a good idea. With pruning 15, 100k is already a very strict limit. At least this is what I interpret from:
```
zless /work/common/asr/librispeech/data/sisyphus_export_setup/alias/baselines/librispeech/ls100/gmm/common_baseline/recognition/train-clean-100-vtln+sat/recog_train-clean-100-vtln+sat-dev-other-ps1.00-lm25.00-iter10-optlm/output/search.log.*.gz  | grep "states after pruning" -A 3 | grep max | grep -o "[0-9]*" | sort -h
```